pkg/manifests/common.go

```diff
@@ -66,6 +66,11 @@ func Namespace(conf *config.Config) *corev1.Namespace {
 		},
 	}
 
+	if !conf.DisableOSM {
+		ns.ObjectMeta.Annotations["openservicemesh.io/sidecar-injection"] = "disabled"
```

Comment:
Are there other pods in this namespace besides NGINX that you want sidecar injected? If not, can we remove this?

Reply:
we don't want the sidecar injected on any of them, including nginx (right)?
but yeah I can remove the annotation. Do we also need to specify that we want the sidecar disabled on all our other pods/deployments as well?

Reply:
The absence of the injection annotation from the workloads is sufficient for not injecting the sidecar 👍🏾

Reply:
should I go ahead and remove it from nginx as well? in theory, that would prevent it from restarting when this change rolls out, right?

Reply:
IIUC, removing the annotation from NGINX pod would still trigger a reconciliation from your control plane right?

Reply:
ah wait nevermind - it would still restart because we're removing the annotation from each pod, so I'm fine to leave nginx as is